src/app/(dashboard)/meetings/page.tsx

```
@@ -0,0 +1,27 @@
+import { ErrorBoundary } from "react-error-boundary";
+
+const Page = () => {
+  const queryClient = getQueryClient();
+  void queryClient.prefetchQuery(trpc.meetings.getMany.queryOptions({}));
+
+  return (
+    <HydrationBoundary state={dehydrate(queryClient)}>
+      <Suspense fallback={<MeetingsViewLoading />}>
+        <ErrorBoundary fallback={<MeetingsViewError />}>
+          <MeetingsView />;
+        </ErrorBoundary>
```

Biome (1.9.4)

[error] There is a suspicious semicolon in the JSX element.

This is usually the result of a typo or some refactor gone wrong.
Remove the semicolon, or move it inside a JSX element.

(lint/suspicious/noSuspiciousSemicolonInJsx)